NitroxModel-Subnautica/DataStructures/GameLogic/CyclopsModel.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Util;
+using ProtoBufNet;
+using UnityEngine;
+
+namespace NitroxModel_Subnautica.DataStructures.GameLogic
+{
+    [Serializable]
+    [ProtoContract]
+    public class CyclopsModel : VehicleModel
+    {
+        [ProtoMember(1)]
```

Comment:
Can we start with 1?  Won't this overlap with items in VehicleModel?

Reply:
The integers are independent. So its no problem see [here](https://nirmalyabhattacharyya.com/2013/09/10/protobuf-net-and-inheritance-hierarchies/)